src/main/java/Brad/Tasks/List.java

```diff
@@ -0,0 +1,133 @@
+package Brad.Tasks;
+
+import java.util.ArrayList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import Brad.Exceptions.dataCorruptedException;
+
+public class List {
```

Comment:
Consider a better name here. List is quite ambiguous and conflict with a java built-in interface, Maybe "Tasklist"